test/cmdlineTests/optimizer_BlockDeDuplicator/output

```diff
@@ -23,14 +23,12 @@ EVM assembly:
   iszero
   tag_5
   jumpi
-  0x00
-  dup1
-  revert
+  revert(0x00, 0x00)
 tag_5:
   pop
-  jump(tag_6)
-    /* "optimizer_BlockDeDuplicator/input.sol":138:174  function f() public { true ? 1 : 3;} */
-tag_2:
+  tag_6
+    /* "optimizer_BlockDeDuplicator/input.sol":77:103  function fun_x() public {} */
+tag_4:
   jump	// out
```

Comment:
This is probably an example of an optimization that is valid, but confusing for tooling, since the original ``jump(tag_6)`` has now become a "jump out"... that's a bit annoying, the optimization is nice, but the resulting jump is neither an ordinary jump nor a jump out in all cases anymore... with ethdebug we could probably annotate this, but without it, it's tricky... maybe we need to just not do this in this case for now (need to get back to look into what actually triggers the optimization in this PR)